sources/packages/backend/libs/src-sql/disbursement-overawards/Bulk-update-bcsl-disbursement-overaward.sql

```diff
@@ -0,0 +1,12 @@
+/*
+ * Update BCSL overawards from SFAS integration to disbursement overawards table.
+ */
+UPDATE	sims.disbursement_overawards disbursement_overawards
+SET		overaward_value = sfas_individuals.bcsl_overaward, 
+		updated_at = NOW(),

```

Comment:
`updated_at` is the inbuilt audit filed right? I think the framework is taking care of it

Reply:
@ann-aot, this query is not using an entity model, which means that TypeOrm has no way to take care of it 😉

Reply:
Not in this case. I did some tests and for raw query that doesn't work.